code/cutscene/movie.cpp

```diff
@@ -205,14 +205,14 @@ void movie_display_loop(Player* player, PlaybackState* state) {
 	auto fpsCapSave = Cmdline_NoFPSCap;
 	Cmdline_NoFPSCap = 1;
 
-	auto lastDisplayTimestamp = timer_get_microseconds();
+	auto lastDisplayTime = timer_get_microseconds();
 	while (state->playing) {
 		TRACE_SCOPE(tracing::CutsceneStep);
 
-		auto timestamp = timer_get_microseconds();
+		auto now = timer_get_microseconds();
 
-		auto passed = timestamp - lastDisplayTimestamp;
-		lastDisplayTimestamp = timestamp;
+		auto passed = now - lastDisplayTime;
```

Comment:
The `now` variable is just a temporary variable to make sure that the time is consistent across all calculations.  Using `currentDisplayTime` could be misleading because it's not being saved, nor is it meaningful in itself.

My only objective in editing this code was to remove the term "timestamp" because these are not timestamps.

Reply:
👍